src/devices/Device-Index.md

```diff
@@ -7,6 +7,7 @@
 * [Adafruit Seesaw - extension board (ADC, PWM, GPIO expander)](Seesaw/README.md)
 * [ADS1115 - Analog to Digital Converter](Ads1115/README.md)
 * [ADXL345 - Accelerometer](Adxl345/README.md)
+* [ADXL357 - Accelerometer](Adxl357/README.md)
```

Comment:
You don't actually need to do that :-) That's done automatically in the build pipeline

Reply:
I removed it again.